### ISSUE TITLE
Audio input not working in Debian 9.

### ISSUE THREAD

**Issue by turtle-bazon** [DragonComputer/Dragonfire#83]
As descibed before, launhcing dragonfire and no reaction on my voice. `dragonfire --cli` seems to work. I check in pavucontrol that have audio sink dragonfire program. And there no problems as I see. But no reaction.

**Comment by mertyildiran** [comment]
@turtle-bazon do you see `YOU: blabla` in the command-line or just literally nothing happens? Because you should say *DRAGONFIRE* or *HEY* to wake her up. If that's not the case:

Have you ever tried to run with `--gspeech` option like: `dragonfire --gspeech`?

**Comment by turtle-bazon** [comment]
Yes, I know it activates via "dragonfire" or "wake up", I undrestood it using `--cli` option. But without `--cli` nothing happens. And yes, I don't see "YOU: blabla" in console.

**Comment by turtle-bazon** [comment]
Can confirm that with `--gspeech` option works near expected.

**Comment by mertyildiran** [comment]
@turtle-bazon troubleshooting this kind of problems is not easy. I don't have much idea about your situation. But I can say Dragonfire uses [Mozilla DeepSpeech](https://github.com/mozilla/DeepSpeech) when `--gspeech` option not plugged in. So I can recommend that try to use Mozilla DeepSpeech alone and see what happens.
